packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -40,4 +44,4 @@ AppConfig.restore = () => {}
 AppConfig.extraGetPropsArgs = () => {}
 AppConfig.freeze = () => {}
 
-export default AppConfig
+export default withReactQuery(withLegacyGetProps(AppConfig))
```

Comment:
Do we need to wrap AppConfig with `withLegacyGetProps`? I don't think we are gonna use it in the test project at all